src/taco-dependency-installer/dependencyInstaller.ts

```diff
@@ -511,7 +517,8 @@ module TacoDependencyInstaller {
                         utilHelper.quotesAroundIfNecessary(elevatedInstallerPath),
                         utilHelper.quotesAroundIfNecessary(self.installConfigFilePath),
                         self.parentSessionId,
-                        utilHelper.quotesAroundIfNecessary(DependencyInstaller.socketPath)
+                        utilHelper.quotesAroundIfNecessary(DependencyInstaller.socketPath),
+                        TacoGlobalConfig.acceptPrompts ? "true" : "false"
```

Comment:
Might want to add a note that we pass the value along on the command line even though it is set in an environment variable because environment variables are not maintained through process elevation.